packages/fyndiq-component-alert/src/index.js

```diff
@@ -53,18 +57,15 @@ export default class Alert extends React.Component {
 
   handleCloseClick() {
     this.close()
-
     if (this.props.onClose) {
       this.props.onClose()
     }
   }
 
   render() {
-    if (this.state.count < 5) {
-      console.log(this.state.count)
-      saveState(this.state.count + 1)
-      const { children, type, unclosable } = this.props
+    const { children, type, unclosable } = this.props
 
+    if (this.state.count < this.props.stopShowingWhenClosed || this.props.stopShowingWhenClosed === undefined) {
```

Comment:
You can unwrap `stopShowingWhenClosed` info in the `const {...} = this.props` line above